app/boards/arm/dao/dao.dtsi

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2023 ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+#include <dt-bindings/zmk/matrix_transform.h>
+
+
+/ {
+	model = "dao";
+	compatible = "dao";
+
+	chosen {
+		zephyr,code-partition = &code_partition;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zmk,kscan = &kscan0;
+		zmk,battery = &vbatt;
+	};
+
+	default_transform: keymap_transform_0 {
+		compatible = "zmk,matrix-transform";
+		columns = <12>;
```

Comment:
I guess this should be 14 columns? (This property currently used by ZMK but might be in the future.)

Reply:
Fixed.